syncto/tests/test_authentication.py

```diff
@@ -107,3 +107,89 @@ def test_in_case_decryption_fails_it_should_raise(self):
                     self.request.registry.cache, 'get',
                     return_value=tempered_cache):
                 self.assertRaises(CryptoError, build_sync_client, self.request)
+
+    def test_uses_ttl_from_settings_if_assertion_has_no_expiration(self):
+        # This assertion comes from PyBrowserID tests cases.
+        # https://github.com/mozilla/PyBrowserID
+        assertion = """
+        eyJjZXJ0aWZpY2F0ZXMiOlsiZXlKaGJHY2lPaUpTVXpFeU9DSjkuZXlKcGMzTWlPaUppY
```

Comment:
These are "old-format" browserid assertions, persona.org used to produce things in that format but it hasn't for a long time and IIUC nothing else uses them.  So it should be safe to ignore it.